### DISCUSSION TITLE
sendSignedTransaction()

### DISCUSSION THREAD

**Discussion by Bhanu-Partap** [ethereum/web3.js#7249] [Q&A]
```
 const tx = await web3.eth.sendSignedTransaction(
    signedTx.rawTransaction,
  );
  ```
  this function is not currently running is there any other alternative for this as I am using the erc20permit so after signing the transaction I am not able to send it .

**Comment by TatyOko28** [comment]
If sendSignedTransaction() is not working, you should:
 Check if the signed transaction is valid before sending it.
 Use eth.sendTransaction() as an alternative if you can sign the transaction in a different way.
 Ensure that signedTx.rawTransaction is correctly formatted and includes gas, nonce, and to fields.

**Reply by Bhanu-Partap** [reply]
Actually i have to create the data of the transaction according to the erc20permit (EIP-712) signing standard .
